### PR TITLE
Add file tags to protect in Housekeeper for Balsamic analysis

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

We need to decide which Balsamic files are needed to keep in Housekeeper for longer than 1 year. The other files will be deleted.

### Added

- Tags to keep

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/constants/housekeeper_tags.py [hunk 1]
```diff
@@ -64,8 +64,37 @@ class GensAnalysisTag:
...
+        "cnv-report",
+        "cnvkit",
+        "delly",
+        "germline",
+        "svdb",
+        "tnscope",
+        "tnscope-umi",
+        "vardict",
+        "vcf2cytosure",
+    ]
```
Comment (ivadym) [inline cg/constants/housekeeper_tags.py:90]: I opted for a general list, but let me know if you think it's better to have more specific tags... It's not extra work as I have the lists already at hand

**Comment by henrikstranneheim** [comment]
Looks good 👍 ! I cannot approve via GitHub since I opened this PR.

**Review by ivadym — APPROVED** [review]
Thanks Henrik! Approving and merging it 😄 

(I just realised that I missed the typing List[List[str], so i added a new commit to fix it)

**Comment by ivadym** [comment]
Deployed

Screenshot 2023-03-15 at 10 06 20

**Pull request merged**